fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/JavaBatchSchemaGenerator.java

```diff
@@ -142,23 +142,26 @@ VALUE VARCHAR(255),
      */
     public void addJobParameterTable(PhysicalDataModel model) {
         // The default for the value is 255, and chosen to update to 4096
-        Table parameter = Table.builder(schemaName, JOBPARAMETER_TABLE)
+        Table table = Table.builder(schemaName, JOBPARAMETER_TABLE)
                 .addVarcharColumn(NAME, 255, NULL) // NAME VARCHAR(255)
                 .addVarcharColumn(VALUE, 4096, NULL) // VALUE VARCHAR(255) (updated to 4096)
                 .addBigIntColumn(FK_JOBEXECID, NOT_NULL) // FK_JOBEXECID BIGINT
                 .addForeignKeyConstraintAltTarget("JBPRMETERFKJBXECID", schemaName, JOBEXECUTION_TABLE, JOBEXECID, FK_JOBEXECID)
                 .addIndex("JP_FKJOBEXECID_IX", FK_JOBEXECID)
                 .addPrivileges(generateGroupPrivilege())
                 .build(model);
-        model.addTable(parameter);
-        model.addObject(parameter);
+
+        table.addTag(FhirSchemaGenerator.SCHEMA_GROUP_TAG, FhirSchemaGenerator.FHIRDATA_GROUP);
```

Comment:
should we use a different group id for the jbatch schema objects?

Reply:
see the following comment about PhysicalDataModel